client/stylesheets/_modal.scss

```diff
@@ -902,6 +902,19 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	height: 100px;
 }
 
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"], .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"] {
+	padding: 11px 50px !important;
+	color: #fff !important;
+	background-color: #f8504b;
+	border: none;
+	border-radius: 4px !important;
+}
+
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"]:hover, .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"]:hover {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector `modFeedbackInitial` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line